contract/tests/test_contract.py

```diff
@@ -269,11 +275,11 @@ def test_contract_onchange_product_id_name(self):
                                     ]))
 
     def test_contract_count(self):
-        """It should return contract count."""
-        count = self.partner.contract_count + 2
+        """It should return sale contract count."""
+        count = self.partner.sale_contract_count + 2
         self.contract.copy()
         self.contract.copy()
-        self.assertEqual(self.partner.contract_count, count)
+        self.assertEqual(self.partner.sale_contract_count, count)
```

Comment:
Please add an test for purchase_contract_count , ideally on the same partner to have different counts